src/Console/Deployer.php

```diff
@@ -127,26 +127,15 @@ private function generateArchive(SymfonyStyle $io, ProgressBar $progress) : void
 
         // Cache PHP's binary in `.bref/bin/php` to avoid downloading it
         // on every deploy.
-        /*
-         * TODO Allow choosing a PHP version instead of using directly the
-         * constant `PHP_TARGET_VERSION`. That could be done using the `.bref.yml`
-         * config file: there could be an option in that config, for example:
-         * php:
-         *     version: 7.2.2
-         */
+        $phpVersion = $projectConfig['php']['version'] ?? DEFAULT_PHP_TARGET_VERSION;
```

Comment:
I think this constant does not exist?

Reply:
I don't know why this change were not there. Anyway, I fixed it in c844d6e